corehq/apps/hqwebapp/static/hqwebapp/js/knockout_bindings.ko.js

```diff
@@ -260,19 +260,24 @@ hqDefine("hqwebapp/js/knockout_bindings.ko", ['jquery', 'knockout', 'jquery-ui/u
                 $(this).parent().parent().addClass("moving").siblings().removeClass('moving');
 
                 var row = $(this).parent().parent();
-                var currentIndex = row[0].attributes['data-order'].value;
-
-                var lastSelectedRowIndex = null;
-                for (var i = 0; i < list().length; i++) {
-                    if (list()[i].selected()) {
-                        lastSelectedRowIndex = i;
+                var currentIndex = parseInt(row[0].attributes['data-order'].value);
+                if (list()[currentIndex].selected()) {
+                    var lastSelectedRowIndex = null;
+                    for (var i = 0; i < list().length; i++) {
```

Comment:
not a big deal, but if you looped through the list backwards, you could break out of the loop when you find a single one that matches, which means you don't have to check every element.